content/python/concepts/built-in-functions/terms/next/next.md

```diff
@@ -0,0 +1,85 @@
+---
+Title: 'next()'
+Description: 'Returns the next element/value from an iterable object.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Collections'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+## **Goal** :dart:
+
+This entry tries to facilitate your comprehension of the Python `next()` function and how to use it properly to get the next item from an iterator object.
+
+## **Why next() function in Python?**
+
+It is a tool on the iterator that allows you to sequentially go through each element of it, using a common method included in every countable set named `iter()`.
+
+## **You need to know (maybe)**
+
+What is an **iterator**? An object that can contain any set of countable values of objects like [_lists, tuples, dictionaries, sets, strings_], traversing all of them if required.
```

Comment:
```suggestion
```

Much of this information isn't needed for this specific entry (like the part about the separate `iter()` function).